signer/src/config/default.toml

```diff
@@ -18,17 +18,18 @@
 # !! Emily API Configuration
 # !! ==============================================================================
 [emily]
-# The URI(s) of the Emily API server to connect to.
+# The endpoint(s) of the Emily API server to connect to.
 #
 # You may specify multiple Emily API servers if you have them. They will be
 # tried round-robin until one succeeds.
 #
-# Format: ["http(s)://<host>:<port>", ..]
+# Format: [ { endpoint = "http(s)://<host>:<port>", api_key = "1234567890abcdef" }, ..]
 # Default: <none>
 # Required: true
 # Environment: SIGNER_EMILY__ENDPOINTS
+# Environment Example: '{"endpoint":"https://api.emilyexample.com","api_key":"1234567890abcdef"},..'
 endpoints = [
-    "http://localhost:3031"
```

Comment:
Hmm, I think I would have preferred to use URI syntax here instead, either authentication (like `http://123123@localhost:3031`) or querystring (`?api_key=123123`), both of which can be extracted from `url::Url`.

Reply:
Hm, that seems odd because it implies something about the structure of the underlying API and the way it expects keys. Why would you prefer this?

Reply:
I think going forward if there's a preference for the way something should be done we should make sure to add that to the ticket, otherwise I'd think all implementations (within reason) should be fair game. This to me looks like a style nit, but if you feel strongly I can go back and change it.